test/system/710-kube.bats

```diff
@@ -135,4 +135,45 @@ status  | =  | null
     run_podman rmi $(pause_image)
 }
 
+@test "podman kube generate - deployment" {
+    skip_if_remote "containersconf needs to be set on server side"
+    local pname=p$(random_string 15)
+    local cname1=c1$(random_string 15)
+    local cname2=c2$(random_string 15)
```

Comment:
Thank you for prefixing with c1/c2. Should the test ever fail, this sort of thing makes it soooo much easier to read the error output!